attestation-service/verifier/src/az_snp_vtpm/mod.rs

```diff
@@ -134,14 +134,13 @@ fn verify_snp_report(
 mod tests {
     use super::*;
 
-    const REPORT: &[u8; 2048] = include_bytes!("../../test_data/az-snp-vtpm/hcl-report.bin");
-    const SIGNATURE: &[u8; 256] = include_bytes!("../../test_data/az-snp-vtpm/tpm-quote.sig");
-    const MESSAGE: &[u8; 122] = include_bytes!("../../test_data/az-snp-vtpm/tpm-quote.msg");
+    const REPORT: &[u8; 3848] = include_bytes!("../../test_data/az-snp-vtpm/hcl-report.bin");
+    const QUOTE: &[u8; 1362] = include_bytes!("../../test_data/az-snp-vtpm/quote.bin");
     const REPORT_DATA: &[u8] = "challenge".as_bytes();
 
     #[test]
     fn test_verify_snp_report() {
-        let hcl_report = HclReport::new(REPORT.to_vec()).unwrap();
+        let hcl_report = HclReport::new(bincode::deserialize(REPORT).unwrap()).unwrap();
```

Comment:
the HCL report should be just bytes, why do we need to deserialize `REPORT`?

Reply:
Yep fixed this.